Community/kavSimulations/devices/kav_rudder.device.json

```diff
@@ -45,6 +50,11 @@
         "id": 5,
         "label": "Set value w/ 'R'",
         "description": "$ will set the value of the display with the 'R' character"
+      },
```

Comment:
Message 4,5,6 - what are they for exactly? could this not be achieved by using msg:0 and msg:3?

Reply:
Hmhm, depends how the config is set up.
Most likely this must be changed too.